src/main/java/io/github/thebusybiscuit/sensibletoolbox/blocks/AngelicBlock.java

```diff
@@ -81,6 +81,11 @@ public void onInteractItem(PlayerInteractEvent event) {
         event.setCancelled(true);
     }
 
+    private boolean isWithinWorldBounds(Block b) {
+        Location loc = b.getLocation();
+        return loc.getY() > loc.getWorld().getMinHeight() && loc.getY() < loc.getWorld().getMaxHeight();
```

Comment:
`getMinHeight()` does not exist on older versions.

Reply:
Thanks, sorted and retested